### ISSUE TITLE
add set_network helper fx, matching get_network

### ISSUE THREAD

**Issue by smjenness** [EpiModel/EpiModel#390]
then use in the modules

set_network should have at the least an argument `network` for network position

**Comment by AdrienLeGuillou** [comment]
Currently `get_network` is a function that extract a network from either a `netsim` or a `netdx` object, so outside of `netsim`.

If I understand correctly, the goal would be to have these functions to work on the `dat` object. 

The way to go would be to have S3Methods `get_network` and `set_network` that do different things on `netsim`, `netdx` and `dat` object.

To do so we would need to set a new class to the `dat` object. This can be done in one line at the end of the exisiting `create_dat_object` function. 

@smjenness what do you think? 
If this route suits you I will need your input for naming the new class.

**Comment by smjenness** [comment]
Ahh, I see. I had not considered that. Maybe let's table this for now, as I don't want to introduce this new class before this release. But down the road, it makes sense that `dat` should be have a class (maybe `netsim_dat`?).

**Comment by AdrienLeGuillou** [comment]
`netsim_dat` seems good. 
We should then call it this way in all the documentation for consistency. I can work on that after the 2.2.0 release

**Comment by chad-klumb** [comment]
I introduced `make_sim_network` and `set_sim_network` in the `multinets` PR for convenient handling of multiple networks and `networkLite`/`networkDynamic` with the same code.  These could serve as the `netsim_dat` class methods for `get_network` and `set_network` if we made generic functions with those names.

Is there any reason to hold off on classing the `dat` object in `create_dat_object` now?
